api-2.0/src/test/java/org/openmrs/module/fhir/api/ConditionServiceTest.java

```diff
@@ -64,4 +89,50 @@ public void getConditionByObsId_shouldReturnMatchingFHIRCondition() {
 		assertEquals(fhirCoding.getCode(), problemAddedObs.getConcept().getUuid());
 		assertEquals(fhirCoding.getDisplay(), problemAddedObs.getConcept().getName().getName());
 	}
+
+	@Test
+	public void createFHIRCondition_shouldReturnCreatedFHIRCondition() {
+		Condition condition = new Condition();
+		String uuid = "75f5b373-5065-11de-80cb-001e378eb67f";
+		IdType id = new IdType();
+		id.setValue(uuid);
+		condition.setId(id);
+		Patient patientRef = Context.getPatientService().getPatient(2);
+		Reference patient = FHIRUtils.buildPatientOrPersonResourceReference(patientRef);
+		condition.setSubject(patient);
+		condition.setClinicalStatus(Condition.ConditionClinicalStatus.ACTIVE);
+		condition.setCode(new CodeableConcept().addCoding(
+				new Coding().setCode("999").setDisplay("NO").setSystem(FHIRConstants.OPENMRS_URI)).setText("NO"));
+
+		Condition resCondition = getService().createFHIRCondition(condition);
+		assertNotNull(resCondition);
+		assertEquals(resCondition.getId(), id.getValue());
+		assertEquals(resCondition.getSubject().getReference(), patient.getReference());
+		assertEquals(resCondition.getCode().getCoding().get(0).getDisplay(), "NO");
+	}
+
+	@Test
+	public void getCondition_shouldReturnSavedCondition() {
+		Context.getAdministrationService().saveGlobalProperty(new GlobalProperty(FHIRConstants
+				.CONCEPTS_CONVERTABLE_TO_CONDITIONS_STORED_AS_OBS, GLOBAL_PROPS_CONDITION_MAPPING_CONCEPT_ID));
+		String patientUuid = "61b38324-e2fd-4feb-95b7-9e9a2a4400df";
+		String conceptUuid = "4a5048b1-cf85-4c64-9339-7cab41e5e364";
```

Comment:
Did you take a look at our naming conventions for constants? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Do you even need these variables?

Reply:
Those constants were already there. Just called/refer to my code.
https://github.com/openmrs/openmrs-module-fhir/blob/master/api/src/main/java/org/openmrs/module/fhir/api/util/FHIRConstants.java
Perhaps I should rename

Reply:
Did you look at our naming conventions for constants?